package.json

```diff
@@ -59,8 +59,8 @@
     "mocha-eslint": "2.0.2",
     "mocha-jsdom": "1.1.0",
     "nyc": "6.4.0",
-    "react-addons-test-utils": "0.14.7",
-    "redux-devtools": "3.1.1",
+    "react-addons-perf": "15.0.1",
+    "react-addons-test-utils": "15.0.1",
     "rimraf": "2.5.2",
```

Comment:
These include upgrades to React and friends to 0.15 (and many required updates to be in sync)